.env

```
@@ -0,0 +1,3 @@
+POSTGRES_URL=postgresql://postgres:postgres123@postgres:5432/ecommerce_inventory
+MONGO_URL=mongodb://admin:admin123@mongodb:27017
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MONGO_URL key should go before the POSTGRES_URL key

(UnorderedKey)